XCUITests/HistoryTests.swift

```diff
@@ -227,4 +231,18 @@ class HistoryTests: BaseTestCase {
         XCTAssertFalse(app.cells.staticTexts["Recently Closed"].isSelected)
         waitForNoExistence(app.tables["Recently Closed Tabs List"])
     }
+
+    func testClearRecentHistory() {
+        navigator.performAction(Action.ClearLastHourHistory)
+        for label in allHistoryEntrys {
+            XCTAssertTrue(app.tables.cells.staticTexts[allWebpages[label]!].exists)
+        }
+        navigator.openURL("google.com")
+        navigator.goto(HomePanel_History)
+        XCTAssertTrue(app.tables.cells.staticTexts["Google"].exists)
+        navigator.performAction(Action.ClearTodayHistory)
```

Comment:
And comment about removing recent history...the older is not removed but the just created is...